pkg/app/context.go

```diff
@@ -735,6 +735,13 @@ func (ctx *RequestContext) GetInt64(key string) (i64 int64) {
 	return
 }
 
+func (ctx *RequestContext) GetUint64(key string) (ui64 uint64) {
```

Comment:
I think the Get*** methods need to be added and should be well commented with a single test.